packages/documentation/.storybook/preview.ts

```diff
@@ -81,6 +83,10 @@ const preview: Preview = {
             title: 'View full screen',
             onClick: openFullScreenDemo,
           },
+          {
+            title: 'Copy variation',
```

Comment:
I think this is a bit clearer, variation could also mean "copy the code for this variation". It could also be "Copy link to current config", but it starts to get a bit long.

```suggestion
            title: 'Copy link',
```